src/crimson/common/formatter.cc

```diff
@@ -55,10 +37,4 @@ ostream& operator<<(ostream& out,
   return out << fmt::format("{}", t);
 }
 
-ostream& operator<<(ostream& out,
-                    const ceph::coarse_real_clock::time_point& t)
-{
-  return out << fmt::format("{}", t);
-}
```

Comment:
so we just need to remove one of the specialization to break the recursion loop, right?

Reply:
That was my understanding.  That particular call stack sequence repeated until the process crashed.